packages/server/lib/server.ts

```diff
@@ -112,6 +112,7 @@ app.route('/connection').post(apiAuth, connectionController.createConnection.bin
 app.route('/environment-variables').get(apiAuth, environmentController.getEnvironmentVariables.bind(connectionController));
 app.route('/sync/deploy').post(apiAuth, syncController.deploySync.bind(syncController));
 app.route('/sync/deploy/confirmation').post(apiAuth, syncController.confirmation.bind(syncController));
+app.route('/sync/frequency').put(apiAuth, syncController.updateFrequency.bind(syncController));
```

Comment:
About the path `/sync/frequency`, this would give me the impression that I can update the frequency for the whole sync (for all connections), which is not the case. 

I would be more explicit with something like: `/sync/updateConnectionFrequency` or `/sync/{sync-name}/connection/{connection-id}`

Reply:
`PUT /sync/update-connection-frequency` to still keep it consistent with how the other endpoints work with taking in `connection_id` and `provider_config_key` as query params

Reply:
Changed 👌🏻